plugins/carbonetes-scan.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: carbonetes-scan
+spec:
+  homepage: https://github.com/carbonetes/kubectl-carbonetes-scan
+  shortDescription: Seamlessly integrates complete container analysis directly into your Kubernetes cluster.
+  version: v1.0.0
+  description: |
```

Comment:
by examining your source code I can tell there's a huge caveat in this plugin: This plugin requires Docker on your machine to work.

Make use of `caveats:` section and check if `docker` is installed, otherwise error properly please.

Reply:
also, what's the reason why you don't distribute the `carbonetes` binary natively for linux/mac and just let users use that? what's the point of docker-run in this case?

Reply:
The reason why we used docker-run is that we already have an existing docker image on the docker hub, which we were currently using on our plugins on other software development tools. We want to maintain these plugins simultaneously when we have some new features, so we've decided to use the same image for creating other plugins.